sdks/java/io/cdc/build.gradle

```diff
@@ -51,8 +51,10 @@ dependencies {
     compile "org.apache.kafka:connect-json:2.5.0"
 
     // Debezium depenedencies
-    compile group: 'io.debezium', name: 'debezium-core', version: '1.3.0.Final'
-    compile group: 'io.debezium', name: 'debezium-connector-mysql', version: '1.3.0.Final'
+    compile group: 'io.debezium', name: 'debezium-core', version: '1.3.1.Final'
+    compile group: 'io.debezium', name: 'debezium-connector-mysql', version: '1.3.1.Final'
+    compile group: 'io.debezium', name: 'debezium-connector-sqlserver', version: '1.3.1.Final'
+    compile group: 'io.debezium', name: 'debezium-connector-postgres', version: '1.3.1.Final'
```

Comment:
all of these debezium dependencies - they are only required for tests, right? if so, let's change them to `compileTest` dependencies instead of `compile`.

Reply:
Only `debezium-core` is required for compilation in `DebeziumSDFDatabaseHistory`.